BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3304,6 +3305,11 @@ void WindowSettings(int windowID)
                                 BDArmorySettings.ENABLE_HOS = !BDArmorySettings.ENABLE_HOS;
                                 CheatCodeGUI = "";
                             }
+                            else if (CheatCodeGUI.ToLower() == "altitudehack") //until we figure out where to put this
+                            {
+                                BDArmorySettings.ALTITUDE_HACKS = !BDArmorySettings.ALTITUDE_HACKS;
+                                CheatCodeGUI = "";
+                            }
```

Comment:
Not sure if Engineered Ascent will get chosed as one of the rounds for S5, but if it is, we can figure out a RunwayRound number to give it instead